pkg/reconciler/apiserversource/resources/receive_adapter.go

```diff
@@ -45,6 +45,7 @@ type ReceiveAdapterArgs struct {
 	Source        *v1.ApiServerSource
 	Labels        map[string]string
 	SinkURI       string
+	CACerts       string
```

Comment:
We need to set to a a value based on what `AddressableFromDestination..`  returned during the reconciliation.

I'd make this a pointer and only set the env variable (line 156 below) when it's actually set/non empty